src/canvas-extractor/edfi_canvas_extractor/graphql/extractor.py

```diff
@@ -105,23 +135,81 @@ def extract(self, body) -> None:
                     "updated_at": section["updatedAt"],
                     })
 
+            enrollments = course["enrollmentsConnection"]
+            for enrollment in enrollments["nodes"]:
+                if enrollment["state"] in ["active", "invited"]:
+                    users = enrollment["user"]
+                    self.students.append({
+                        "id": users["_id"],
+                        "sis_user_id": users["sisId"],
+                        "created_at": users["createdAt"],
+                        "name": users["name"],
+                        "email": users["email"],
+                        "login_id": users["loginId"],
+                    })
+
+                    self.enrollments.append({
+                        "id": enrollment["_id"],
+                        "user_id": users["name"],
```

Comment:
Small bug here. Should be:
```suggestion
                        "user_id": users["_id"],
```

Reply:
Changes applied